src/components/createConnectDecorator.js

```diff
@@ -13,14 +15,10 @@ export default function createConnectDecorator(React, Connector) {
       render() {
         return (
           <Connector select={state => select(state, this.props)}>
-            {this.renderChild}
+            {stuff => <DecoratedComponent {...stuff} {...this.props} />}
```

Comment:
Probably not.. Since it depends on `this.props`, we have to make it a lambda, or else the props will be stale.